StudyHub/StudyHub/Validators/AuthorizationValidators/ForgotPasswordValidator.cs

```diff
@@ -0,0 +1,15 @@
+﻿using FluentValidation;
+using StudyHub.Common.Requests;
+
+namespace StudyHub.Validators.AuthorizationValidators;
+
+public class ForgotPasswordValidator : AbstractValidator<ForgotPasswordRequest>
+{
+    public ForgotPasswordValidator()
+    {
+        RuleFor(request => request.Email)
+            .NotEmpty()
+            .EmailAddress()
+            .WithMessage("Your email is not real");
```

Comment:
Maybe better "Such email is not exist"

Reply:
this is not relevant for such message. If something not exist, it means it not exist in somewhere, for instance, DB. Based  on the above, I dont think something wrong here